README.md

```diff
@@ -1,2 +1,23 @@
-# student-developer-setup
-Bootstrapped instructions for setting up development environments for student developers
+# Yan Zhuang student-developer-setup
+
+1) Created fork on GitHub
+
+2) Created README.md
+
+3) Pushed to GitHub, created pull request
+
+4) Using Pycharm (Working on Windows System)
```

Comment:
This is just a comment, not a request for changes, but as you're working on Windows are you working inside of WSL2 or are you just working in the Windows terminal on Windows? Either is fine, though if you aren't working in WSL2 you might find that most setup instructions that people use are going to need small revisions to work in Windows Powershell.

Reply:
Sorry for the misleading documents. I run my program on wsl and write code in vscode installed at Windows. I've installed wsl long time ago so I didn't document this step.